client/editSchedule.js

```diff
@@ -1,9 +1,11 @@
 var editingRoundReact = new ReactiveVar(null);
 
 Template.editSchedule.helpers({
+  competition: function() {
+    return Competitions.findOne(this.competitionId);
+  },
   unscheduledRounds: function() {
-    var rounds = getUnscheduledRounds(this.competitionId);
-
+    var rounds = getScheduledRounds(this.competitionId, false);
```

Comment:
Ah! You caught me. I was not pleased with this. In python, I do this occasionally, but it's more readable because you can say `getScheduledRounds(competitionId, scheduled=False)`. I toyed around with breaking it up into the following:

```javascript
var scheduled = false;
getScheduledRounds(competitionId, scheduled);
```

Still not great, though... the dictionary solution you suggested is a cute way of achieving the python like "keyword args" syntax.

Agreed that a typechecked enum like what Java gives you is *awesome*.

Renaming to `getRounds` seems good, you're right that the word "scheduled" in the name is not helpful.